CorsixTH/Lua/hospital.lua

```diff
@@ -167,15 +167,8 @@ function Hospital:Hospital(world, avail_rooms, name)
   self.policies = {}
   self.discovered_diseases = {} -- a list
 
-  self.discovered_rooms = {} -- a set; keys are the entries of TheApp.rooms, values are true or nil
-  self.undiscovered_rooms = {} -- NB: These two together must form the list world.available_rooms
-  for _, avail_room in ipairs(avail_rooms) do
-    if avail_room.is_discovered then
-      self.discovered_rooms[avail_room.room] = true
-    else
-      self.undiscovered_rooms[avail_room.room] = true
-    end
-  end
+  -- Populate the level's rooms available to this hospital, including their discovery status
+  self.room_discovery = avail_rooms
```

Comment:
Document the structure of the variable, what fields are expected to exist?
For an example, see world.lua line 102 `self.next_earthquake`